backend/infrahub/webhook/tasks.py

```diff
@@ -132,7 +135,14 @@ async def configure_webhooks(service: InfrahubServices) -> None:
 
         deployment_id_webhook_trigger = deployments[WEBHOOK_TRIGGER.name].id
 
-        webhook_configure_automation = await client.find_automation(id_or_name=AUTOMATION_NAME_RUN)
+        # Reading all automations here as a workaround for
+        # https://github.com/PrefectHQ/prefect/issues/16822
+        automations = await client.read_automations()
```

Comment:
could we use the function `read_automations_by_name` directly ?

Reply:
I think the filter can make a significant difference if there is a lot of automation configured on the server